tensorflow/tools/ci_build/linux/ppc64le/cpu/run_cc_core.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
For our scripts, I split them up to speed up the presubmits.
However you may want to merge them, as separate scripts will need to redundantly build everything.
So you may simply have two scripts, "run_py2.sh" and "run_py3.sh" and they can simply build and test "//tensorflow/... -//tensorflow/compiler/..."

Reply:
Thanks for the suggestion, I will update the code to run this way.